### PR TITLE
Hide values in "reshuffle env --list", and quote them on display

### PR DESCRIPTION
**Pull request by arielshaqed**
1. By default `reshuffle env --list` shows all values as `<hidden>`.
2. To include values, `reshuffle env --list --include-values`.
3. Both `--list` or `--get` quote values, allowing a single
   interpretation for `<hidden>`, `undefined`, and `"value"`.

### CODE REVIEW

File: cli/src/commands/env.ts [hunk 1]
```diff
@@ -100,11 +118,15 @@ export default class Env extends Command {
     }
   }
 
-  protected async list(appId: string) {
-    const { variables } = await this.lycanClient.getEnv(appId, null);
-    // Output in sorted order (of variable names).
-    const sorted = variables.map(vvToString).sort();
-    this.log(sorted.join('\n'));
+  protected async list(appId: string, includeValues: boolean) {
+    try {
```
Comment (vogre) [inline cli/src/commands/env.ts]: No need for `try-catch` - I think errors are handled at a different level.

File: cli/src/commands/env.ts [hunk 2]
```diff
@@ -100,11 +118,15 @@ export default class Env extends Command {
...
+      const { variables } = await this.lycanClient.getEnv(appId, null);
+      // Output in sorted order (of variable names).
+      const sorted = (includeValues ? variables : eraseValues(variables)).map(vvToString).sort();
```
Comment (vogre) [inline cli/src/commands/env.ts]: `sort` without a comparator function is suspicious, although a `=` separator guarantees that we are sorting by variable names and not the whole value.

File: cli/src/commands/env.ts (hunk 1 above)
Comment (arielshaqed) [reply cli/src/commands/env.ts]: Yuck, that's debugging code.  Squashing it out at the source (and removing this stain on code history).

File: cli/src/commands/env.ts (hunk 2 above)
Comment (arielshaqed) [reply cli/src/commands/env.ts]: Indeed, the output string itself is sorted.

**Comment by arielshaqed** [comment]
Yup, I know about these weirdo snaps.

Thanks!  Publishing this once tests pass...

**Pull request merged**